### DISCUSSION TITLE
2FA with OpenID - Wordfence

### DISCUSSION THREAD

**Discussion by loelismurf** [oidc-wp/openid-connect-generic#569] [Q&A]
Dear Community,
i have installed the free version of "Wordfence" and enabled the 2FA it works very well.
Now if some one log in with OAUTH in the OpenID Connect Generic Client, no 2FA prompt are displayed, is there a possability to force it that it not bypass the 2FA Login?

Kind Regards

**Comment by timnolte** [comment]
The purpose of using OpenID Connect in place of the WordPress login is that it is your authentication and identity provider. You should enable MFA in your identity provider.

**Reply by loelismurf** [reply]
We are an association that has an umbrella organisation, the umbrella organisation provides a member database that also manages the login. We cannot demand that 2FA is mandatory, so we would have requested it on our website.
